package com.dmg.movieticket.dto.request;

import jakarta.validation.constraints.DecimalMax;
import jakarta.validation.constraints.DecimalMin;
import jakarta.validation.constraints.Min;
import jakarta.validation.constraints.NotNull;

import java.math.BigDecimal;

public record CreateRefundPolicyRequest(

        @NotNull
        @Min(0)
        Integer hoursBeforeShow,

        @NotNull
        @DecimalMin("0.00")
        @DecimalMax("100.00")
        BigDecimal refundPercentage

) {
}